src/inc/msquic.h

```diff
@@ -399,6 +399,7 @@ typedef struct QUIC_STATISTICS {
     uint32_t Rtt;                       // In microseconds
     uint32_t MinRtt;                    // In microseconds
     uint32_t MaxRtt;                    // In microseconds
+    uint32_t ChangeSrcCidCount;         // Number of times the source CID changed.
```

Comment:
Don't change `QUIC_STATISTICS`.

Reply:
Okay, then should I get statistics via v2?

Reply:
Just v2. v1 is done.